aws/rust-runtime/aws-sig-auth/src/query_writer.rs

```diff
@@ -0,0 +1,100 @@
+/*
+ * Copyright Amazon.com, Inc. or its affiliates. All Rights Reserved.
+ * SPDX-License-Identifier: Apache-2.0.
+ */
+
```

Comment:
Doesn't look like this module is referenced anywhere?

Reply:
good catch, result of a bad `git add`